### ISSUE TITLE
disk.usage does not work on AIX

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt#29770]
When I try to run the `disk.usage` module on an aix machine I receive a stack trace error.

Tested on AIX OS versions: 6,7
Test on salt versions: 2015.5.1, 2015.8.3

```
ch3ll@ch3ll-centos states]$ sudo salt '*aix*' disk.usage                                                                                                                    
aix-upgrade-ch3ll:
    The minion function caused an exception: Traceback (most recent call last):
      File "/opt/salt/lib/python2.7/site-packages/salt/minion.py", line 1039, in _thread_return
        return_data = func(*args, **kwargs)
      File "/opt/salt/lib/python2.7/site-packages/salt/modules/disk.py", line 91, in usage
        while not comps[1].isdigit():
    IndexError: list index out of range
```

**Comment by Ch3LL** [comment]
ZD-553

**Comment by dmurphy18** [comment]
Reminder - the fix will be to be made to 2015.5 branch and forwarded to 2015.8 branch

**Comment by Ch3LL** [comment]
I've tested and confirmed that this is now working in 2015.8.5. Closing now.
